br/pkg/lightning/config/config_test.go

```diff
@@ -273,6 +273,7 @@ func TestInvalidSetting(t *testing.T) {
 	cfg.TikvImporter.SortedKVDir = "."
 	cfg.TiDB.DistSQLScanConcurrency = 1
 	cfg.Mydumper.SourceDir = "."
+	cfg.TiDB.PdAddr = "234.56.78.90:12345"
```

Comment:
I checked that in master branch test still passes with this line. Is it needed?

Reply:
Hi, please check if this change is needed @mittalrishabh

Reply:
i think i added to fix the CI test failure. Let me try removing it.

Reply:
Yes, this change is required with my changes. Otherwise CI fail. If PD address is not given than it throws invalid tidb address but test case is expecting invalid port address.